pkg/stack_mgr/stack_service.go

```diff
@@ -137,21 +137,17 @@ func (s *StackService) Remove(ctx context.Context, stackName string) error {
 	if err != nil {
 		return errors.Wrap(err, "unable to get a list of stacks")
 	}
-	stackNames := map[string]bool{}
-	for currentStackName := range stacks {
-		if currentStackName != stackName {
-			stackNames[currentStackName] = true
-		}
-	}
 	stackNamesList := []string{}
-	for stackName := range stackNames {
-		stackNamesList = append(stackNamesList, stackName)
```

Comment:
`stackNames` was never used aside from the generation of `stackNamesList`, and this simplifies that by removing the intermediate map